.github/workflows/perf-testing-setup.yml

```diff
@@ -2,12 +2,11 @@ name: Throughput testing workflow
 
 on:
   workflow_dispatch:
-  pull_request:
-    branches: [ dev ]
 
 env:
   TESTS_DIR_PATH: ".ci/perf_tests/k6scripts/"
   PYTHON_VERSION: "3.8"
```

Comment:
Lets use 3.10 or 3.11. whatever the latest available

Reply:
will update